### PR TITLE
packer 0.80

### PR DESCRIPTION
**Pull request by peterloron**
Updated Packer formula to Packer 0.80.

### CODE REVIEW

File: Library/Formula/packer.rb [hunk 1]
```diff
@@ -146,9 +121,10 @@ class Packer < Formula
   # Note that this is *not* the latest commit; the API was changed after this
   # commit and Packer 0.7.5 can't build against the newer version.
   go_resource "golang.org/x/oauth2" do
-    url "https://go.googlesource.com/oauth2.git", :revision => "b3f9a68f05ff3f8b323cd6917f1f237cdbc6edaa"
+   url "https://go.googlesource.com/oauth2.git", :revision => "b3f9a68f05ff3f8b323cd6917f1f237cdbc6edaa"
```
Comment (mistydemeo) [inline Library/Formula/packer.rb]: Don't change whitespace here, please!

File: Library/Formula/packer.rb [hunk 2]
```diff
@@ -146,9 +121,10 @@ class Packer < Formula
...
   end
 
+
```
Comment (mistydemeo) [inline Library/Formula/packer.rb]: Likewise, don't add a newline here please.

File: Library/Formula/packer.rb [hunk 3]
```diff
@@ -146,9 +121,10 @@ class Packer < Formula
   # Note that this is *not* the latest commit; the API was changed after this
   # commit and Packer 0.7.5 can't build against the newer version.
```
Comment (mistydemeo) [inline Library/Formula/packer.rb]: Is this still true? Does Packer 0.8.0 work with a newer oauth2?

**Comment by mistydemeo** [comment]
This doesn't build for me; it looks like dependencies need to be updated:

```
# github.com/mitchellh/packer/template
template/parse.go:208: undefined: mapstructure.StringToTimeDurationHookFunc
```

**Comment by DomT4** [comment]
There's another PR pending for this in #41237.

**Comment by peterloron** [comment]
Hmm. I was able to build it, but I needed to make sure go was uninstalled and I had to clear my cache directory. I was unable to get it to build using the latest oauth2. I did not dig into the root issue.

**Comment by DomT4** [comment]
Closing in favour of #41351.

**Pull request closed without merging**